lib/go-tc/servercheckextensions.go

```diff
@@ -0,0 +1,86 @@
+package tc
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"database/sql"
+	"errors"
+	"regexp"
+
+	"github.com/apache/trafficcontrol/lib/go-tc/tovalidate"
+	"github.com/apache/trafficcontrol/lib/go-util"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+// ServerCheckExtensionNullable represents a server check extension used by Traffic Ops.
+type ServerCheckExtensionNullable struct {
+	ID                    *int       `json:"id" db:"id"`
+	Name                  *string    `json:"name" db:"name"`
+	Version               *string    `json:"version" db:"version"`
+	InfoURL               *string    `json:"info_url" db:"info_url"`
+	ScriptFile            *string    `json:"script_file" db:"script_file"`
+	IsActive              *int       `json:"isactive" db:"isactive"`
```

Comment:
I know it would entail a bunch more work - including either a database migration or a custom scan/marshal - but since this is a new endpoint we could switch this to be an actual boolean.

Reply:
That's fine